src/test/java/com/puppycrawl/tools/checkstyle/AuditEventDefaultFormatterTest.java

```diff
@@ -104,6 +104,23 @@ public void testFormatModuleNameDoesNotContainCheckSuffix() {
     }
 
     @Test
+    public void testFormatModuleWithModuleId() {
+        final AuditEvent mock = PowerMockito.mock(AuditEvent.class);
```

Comment:
Why is this a mock? Why can't we instantiate `AuditEvent` and `LocalizedMessage`?
Mocks should be a last resort when deep code can be done normally.

You were able to get by without a mock in `XMLLoggerTest`.